lsof.adoc

```diff
@@ -0,0 +1,22 @@
+=== sysutils/lsof major upgrade
+
+Links
+link:https://github.com/lsof-org/lsof[lsof project repo] URL: link:https://github.com/lsof-org/lsof[https://github.com/lsof-org/lsof] +
+
+Contact: Larry Rosenman <ler@FreeBSD.org>
+
+sysutils/lsof in ports has had a major upgrade to no longer go looking 
+in /dev/kmem for data and using userspace API's.  This has taken a long
+time to hit the tree, but is finally done.  This fixes lsof(1) to work with
+ZFS again for the first time since 13.0-RELEASE.
+
+This will make the maintenance of sysutils/lsof much easier going forward.
+
+To the kernel folks: if you make changes that break lsof, please submit a 
+github PR to https://github.com/lsof-org/lsof.
```

Comment:
If there's a way to find out about changes breaking lsof, best to mention it here, I think.